### ISSUE TITLE
Issues in deciphering TensorFlowLite Interpreter output

### ISSUE THREAD

**Issue by VirVisha** [tensorflow/tensorflow#50375]
I have been doing an image classification problem where in the objective is to train a predefined neural network model with set of tfrecords and do inference. This all is happening with reasonable accuracy In Colab.

Subsequent to this I converted the saved_model.pb into model.tflite file. I have checked it against the netron app it is seemingly taking correct inputs (which is an image tensor).

After this I called interpreter.invoke().

Following this when I try to decipher the output tensor I should be able to at least render the output the image, but i am having difficulty in doing this.

This is the link of colab notebook https://colab.research.google.com/drive/1kisxBtImUS7sT2aQlsEKfcXaEnsKumGw?usp=sharing where I have maintained the code.

I have other colab notebooks where similar code was done with training for upto 7500 iterations, but i am stuck in all the case at the interpreter level, since i have to port this app on to Android platform

**Comment by xunkai55** [comment]
Could you help clean up the colab a little bit?

It seems you run an object detection TF model somewhere in between, and then work on the TFLite model. The TFLite model outputs a 1x10x4 tensor, so it doesn't look like an image classification model. Is it also an object detection model?

And for both image classification / object detection, the output is not an image, so it doesn't make sense to directly render the output.
 - image classifcation should output the confidence scores for each category.
 -  object detection usually output the relative coordinates of the bounding boxes, and category labels (in another tensor, very likely).

**Comment by chunduriv** [comment]
@VirVisha, Is your issue resolved now? Else, did you checked [this response](https://github.com/tensorflow/tensorflow/issues/50375#issuecomment-872114199)?
